src/components/Filters/components/Input/Input.tsx

```diff
@@ -52,6 +52,7 @@ const Input: React.FC<InputProps> = ({
   maxLength,
   pattern,
   patternMessage,
+  placeholder = 'String',
```

Comment:
Another default placeholder than 'String' or an empty one would be better in this case

Reply:
same with other placeholders: 'Date', 'Number', etc

Reply:
Could you elaborate on that? `String`, `Date`, `Number` etc. are default placeholders defined by design

Reply:
These were probably put there by Mate as labels not real practical placeholders

Reply:
Those are real default placeholders. They're helping to end user to understand what data type can input and if there should be something more specific then developer can define custom placeholder which covers this PR :)